### DISCUSSION TITLE
[keyball61] Shiftキー押下時、一部のキー入力がズレる

### DISCUSSION THREAD

**Discussion by KIUPICHI** [Yowkees/keyball#279] [Q&A]
赤で囲った部分について、shiftを押しながら入力すると
想定より左に一つずつズレた記号が入力されます。

具体的には、
レイヤー0→shift押下時
6→&
7→'
8→(
9→)
0→特に何も入力されない

また、キーマップ上の「MO(1)」を押しながら入力した場合は特にズレもなくキーマップ通りの結果になります。
恐らく、日本語配列とUS配列の違いがこの症状を招いているのではないかと思うのですが解決策がわからず、
ご教授お願いできればと思います。よろしくお願いいたします。[
![スクリーンショット 2023-07-20 115811](https://github.com/Yowkees/keyball/assets/140035694/c6e238f6-bbc7-4b04-a8d1-b6c9301fc72d)
](url)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。
下記画像の緑矢印部の言語をENGLISHに変更するとどうでしょうか？
![remao](https://github.com/Yowkees/keyball/assets/84209772/8c19c75c-ff37-49cb-a716-2ee72a55bc1c)

**Comment by KIUPICHI** [comment]
ご回答ありがとうございます！
言語をENGLISH(US)にし、適当なキーを一つ変更してFLASHしてみたのですが特に変化がありませんでした。

**Comment by Yowkees** [comment]
シフトを押しながら数字キーを押した時の挙動は、ご指摘の通り日本語配列とUS配列の違いで、PC側の設定に依存すると思います。
（本件はKeyballシリーズに限らずQMK_Firmwareを使用するキーボード全般の問題でして、私のわかる範囲で回答させていただきます）

US配列で使用されたいということのようですので、やはりPC側設定をUS配列に設定いただければと思います。
以上、ご検討お願いいたします。
